agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -195,7 +195,7 @@ public static void overlayEnvVars(Configuration config) throws IOException {
 
         if (isTrimEmpty(config.role.instance)) {
             // only use WEBSITE_INSTANCE_ID as a fallback
-            config.role.name = getEnv(WEBSITE_INSTANCE_ID);
+            config.role.instance = getEnv(WEBSITE_INSTANCE_ID);
```

Comment:
can we add a test for this change? a config with role name?

Reply:
see new tests in this PR